Maindest merge issue

Task :app:processDebugManifest FAILED
/home/pradeep/Documents/react/mobile/AwesomeProject/android/app/src/main/AndroidManifest.xml:12:9-17:66 Error:
        Attribute provider#com.whatsapp_stickers.stickers.StickerContentProvider@authorities at AndroidManifest.xml:12:9-17:66 requires a placeholder substitution but no value for <contentProviderAuthority> is provided.

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute provider#com.whatsapp_stickers.stickers.StickerContentProvider@authorities at AndroidManifest.xml:12:9-17:66 requires a placeholder substitution but no value for <contentProviderAuthority> is provided.

--- a/README.md
+++ b/README.md
@@ -20,10 +20,30 @@
   	include ':react-native-whatsapp-stickers-android'
   	project(':react-native-whatsapp-stickers-android').projectDir = new File(rootProject.projectDir, 	'../node_modules/react-native-whatsapp-stickers-android/android')
   	```
-3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
-  	```
-      compile project(':react-native-whatsapp-stickers-android')
-  	```
+3. Insert the following lines inside `android/app/build.gradle`:
+```
+android {
+	...
+	defaultConfig {
+  ...
+		def contentProviderAuthority = applicationId + ".stickercontentprovider"
+		// Creates a placeholder property to use in the manifest.
+		manifestPlaceholders =
+			[contentProviderAuthority: contentProviderAuthority]
+		// Adds a new field for the authority to the BuildConfig class.
+		buildConfigField(
+			"String",
+			"CONTENT_PROVIDER_AUTHORITY",
+			"\"${contentProviderAuthority}\""
+		)
+	}
+}
+...
+dependencies {
+	...
+	compile project(':react-native-whatsapp-stickers-android')
+}
+```
 
 ## Usage
 

